languages/en.ini

```
@@ -740,6 +749,7 @@ Institutional Login = "Institutional Login"
 institutional_login_desc = "Enter your campus-wide username and password."
 Instructor = "Instructor"
 Interlibrary Loan Requests = "Interlibrary Loan Requests"
+interlibrary_loan_requests_text = "Request materials through Inter-library Loan system."
```

If we remove "the" we should also remove "system". I also suggest removing the dash since we don't include it in other references to ILL in the language file.

```suggestion
interlibrary_loan_requests_text = "Request materials through Interlibrary Loan."
```